functions.php

```diff
@@ -31,5 +31,50 @@ function truncate(
     }
 }
 
+/**
+ *Показывает дату публикации поста в относительном формате,
+ * удобном для пользователя. Т.е. вместо конкретной даты и времени
+ * показывает эту же дату в виде количества прошедших с данного момента
+ * минут, часов, дней, недель или месяцев.
+ * @param string $postCreatedData время создания поста
+ * @return string возвращает дату в виде количества прошедших с
+ * данного момента минут, часов, дней, недель или месяцев.
+ */
+
+function getDateDiff(string $postCreatedData): string
+{
+    $dateNow = date_create();
+    $datePostCreated = date_create($postCreatedData);
+    //считает разницу с текущим временем
+    // date_diff функция возвращает объект DateInterval
+    $dateDifference = date_diff($datePostCreated, $dateNow);
```

Comment:
Вместо комментария нужно делать так
```suggestion
    /** @var DateInterval $dateDifference */
    $dateDifference = date_diff($datePostCreated, $dateNow);
```